Gibe.Navigation.Umbraco/UmbracoNavigationProvider.cs

```diff
@@ -32,66 +32,66 @@ public class UmbracoNavigationProvider<T> : INavigationProvider<T> where T : INa
 						typeof(SettingsNodeType), 
 						filters, 
 						navigationElementFactory)
-		{
-		}
+        {
+        }
 
-		public UmbracoNavigationProvider(
+        public UmbracoNavigationProvider(
 				IUmbracoNodeService umbracoNodeService,
 				IUmbracoWrapper umbracoWrapper,
 				INodeTypeFactory nodeTypeFactory,
 				int priority,
 				Type rootNodeType,
 				IEnumerable<INavigationFilter> filters,
 				INavigationElementFactory navigationElementFactory)
-		{
-			_umbracoNodeService = umbracoNodeService;
-			_umbracoWrapper = umbracoWrapper;
-			_nodeTypeFactory = nodeTypeFactory;
-			Priority = priority;
-			_rootNodeType = rootNodeType;
+        {
+            _umbracoNodeService = umbracoNodeService;
+            _umbracoWrapper = umbracoWrapper;
+            _nodeTypeFactory = nodeTypeFactory;
+            Priority = priority;
+            _rootNodeType = rootNodeType;
 			_filters = filters;
 			_navigationElementFactory = navigationElementFactory;
-		}
+        }
 
-		public int Priority { get; }
+        public int Priority { get; }
 
-		public IEnumerable<T> GetNavigationElements()
-		{
-			var topLevel = _umbracoNodeService.GetNode(_nodeTypeFactory.GetNodeType(_rootNodeType));
```

Comment:
Maybe we should make the IPbulishedContent an optional parameter and have it throw an exception if the nodetype you want requires it?

Would be nicer than passing null in if we don't have that information